### ISSUE TITLE
Add feature for telemetric data

### ISSUE THREAD

**Issue by christianlupus** [nextcloud/cookbook#830]
It would be interesting, what versions are installed in the NC world.

Interesting values would be (not necessarily complete list):
- What NC core version are our users running on
- What DB are they using?
- Which cookbook app version are they using?

This might help with certain design decisions during development.

One might have the option to ask at nextcloud GmbH if they already have telemetry data and if they were willing to share that.

---

Depends on #1067

**Comment by christianlupus** [comment]
No, unfortunately not. Maybe someone at the NC core team knows if they have something similar in planning or even yet set up. Then, their measurements might be useful (filtered for the cookbook app).

**Comment by tcitworld** [comment]
There's https://ncpw.mdns.eu/

**Comment by christianlupus** [comment]
We should ask for consent if telemetric data can be submitted. Thus, I added the #1067 dependency.

**Comment by christianlupus** [comment]
Having brought this up in the forum, there was quite some resistance against collecting telemtric data: https://help.nextcloud.com/t/generic-domain-for-app-development/147153

Having read the discussion there, I am not so sure if there a good way to help us while still maintaining the data-saving principles of Nextcloud.

I would like to hear your opinions. From my perspective, that discussion could be settled with the drawback that we will have to make more strict decisions. Let me hear your opinion.
